operations/governance-remediation-process.md

```diff
@@ -0,0 +1,83 @@
+
+In circumstances where the project lacks the people in the defined roles to execute governance processes, the TOC with support from TAG Contributor Strategy serves as an independent advisor to recommend and guide the project to a fair, transparent outcome. In some cases, making specific recommendations to the project to resolve the issue.  If no other course is available, the TOC will directly approve changes to project governance in order to allow it to operate again.
+
+In cases where leaders of an open source project willfully disregard the project's published governance or CNCF principles around open governance, the TOC, with advice from the Governance WG, will prepare a remediation plan. This plan could involve supervised project administration, changes to the written governance, removal of specific project leaders, or whatever other measures are required to restore the project to a state where it is following its own governance rules, and that governance is in accord with CNCF principles. Should there be no way to restore the project to community operation, the TOC will consider archiving the project.
+
+In rare cases, governance failure may be accompanied by possible CoC violations by individual project leaders. Should this happen, the assigned team will engage the CNCF Code of Conduct Committee and the two teams will coordinate their evaluation.
+
+### Out of Scope:
+
+**Conduct Issues**: Handled by the CNCF Code of Conduct [processes](https://github.com/cncf/foundation/blob/main/code-of-conduct.md#reporting). This includes behavior by project leaders which is a violation of the CNCF CoC; such behavior should be reported to the CoCC and not handled through Governance Remediation.
+
+**Technical Disagreements**: Projects should use or establish governance to resolve such disputes independently.
+
+**Lifecycle Processes**: Elements already covered in the project's lifecycle review.
+
+## Reporting Governance Remediation
+
+**Public Reports: **In most cases, project governance issues will be reported publicly to the TOC by filing [an Issue in the TOC repository](https://github.com/cncf/toc/issues) of type Governance. Should a public issue be reported in some other venue (the project's issue tracker, mailing list, etc.) an issue in the TOC repo should be created for tracking and assignment.
+
+**Anonymous Reports**: Should the reporter not wish their name to be known, they should email &lt;TOC-confidential-email-here>.  The TOC team will then create an issue in the TOC repo tracking the remediation. The identity of the anonymous reporter will be known only to the TOC team.
+
+**Confidential Reports**: Should there be risks around the details of the remediation request being public, the reporter will email &lt;TOC-confidential-email-here>.  The TOC and the assessment team will then use privately shared documents to track the status of the remediation request.
```

Comment:
> &lt;TOC-confidential-email-here>

looks like a placeholder

Reply:
Do we know of examples where anonymous reporting would have been helpful, that are not already covered by CoC? I would be a bit worried that this opens up an easy path for malicious actors to create work, and significant mental load, on the TOC and the project maintainers

Reply:
If there are already examples of this, they are not sharable due to their nature -- I can't even confirm whether there are because of that.

However, this anonymous/confidential reporting is set up to parallel the same mechanism we have for the CoC.  There are always cases where a reporter has a reason to fear retaliation from project leadership for raising a governance issue, and requiring them to take that risk means that many governance issues will go unreported until they reach the crisis stage.

Reply:
If the reporter has reason to fear retaliation, doesn't that make it a CoC issue anyway? I'm just trying to avoid the overhead of a duplicate process here. 

If we want this separate process, I do still think there needs to be a requirement on the reporter to show that _someone_ has attempted to resolve through the normal channels within the project

Reply:
Speaking as a member of the CoCC, we are not going to handle reports that a project is, for example, not being vendor neutral.  That's outside the CoC.

There IS considerable overlap between governance remediation and the Technical Leadership Principles, though.

Reply:
@mrbobbytables do we have an email we can add here?

I think providing an anonymous email avoiding allowing these to escalate into CoC incidents because there wouldn't be someone to "retaliate" against. It makes sense to keep these separate.

Reply:
TOC has a private email, which would cover both Governance Remediation and Leadership Principles violation.  That works, particularly given that in most cases where anyone would want confidentiality, they're going to be the same thing (e.g. "Maintainer X is flagrantly disregarding the written governance")

Reply:
I suspect it is not widely known (a) that the CNCF has Technical Leadership Principles (b) that they nominally apply to project maintainers, and not just positions like TAG chair or TOC member. They do not seem to be mentioned during [project onboarding](https://github.com/cncf/toc/blob/5c7fff9ae312eb47ca32087850183d24bfe61b86/.github/ISSUE_TEMPLATE/project-onboarding.md?plain=1#L9).

([This was flagged, but not addressed, at the time](https://github.com/cncf/toc/pull/1063#discussion_r1212329082).)

Reply:
@craigbox i disagree with it not being addressed. I believe my comment in response to your concern did address the requirement imposition as you indicated. 

I agree that this is not widely known. @mrbobbytables would you ensure the project onboarding includes an overview of the principles and the technical leadership principles? It is important that projects understand how the TOC intends to interact with projects, which includes the expectations regarding the conduct of leaders in our projects and groups.

Reply:
Yup - I thought it was part of the onboarding template, but it appears it is not. Will make that change shortly.

Reply:
Thankyou!

Reply:
@TheFoxAtWork Pardon me, I only found this discussion again while looking up the PR that committed the changes to the principles document.  I don't see a follow-up comment from you on #1063.  Was it on another issue?

Reply:
🤦‍♀️ sigh... it was responded too but is suspect it is not actually responded to on the issue: https://github.com/cncf/toc/pull/1063#discussion_r1760881612.

Reply:
> They do not seem to be mentioned during [project onboarding](https://github.com/cncf/toc/blob/5c7fff9ae312eb47ca32087850183d24bfe61b86/.github/ISSUE_TEMPLATE/project-onboarding.md?plain=1#L9).

Independent of that, we have projects that were already onboarded and predate this, I wasn't aware of these either even as a listed maintainer.

I don't see anything remotely objectionable in them, but IMHO if these are meant to apply to project maintainers it hasn't been effectively communicated to project maintainers.

In the future if changes are made to these, or similar policies introduced, an email should probably go out to the maintainers, or something along these lines ...?